cdhweb/static_src/global/components/forms.scss

```diff
@@ -1,7 +1,30 @@
-// Examples
-// .form-input {
-// ...
-// }
-// .form-select {
-//   ...
-// }
+// Input with joined-up inline submit button.
+// Used for search and email signup forms.
+.text-input-and-button-group {
+  --search-control-height: 36px;
+  --radius: 24px;
+  display: flex;
+
+  :where(input) {
+    font: inherit;
+    height: var(--search-control-height);
+  }
+  :where(:is(input[type='text'], input[type='email'])) {
+    flex-grow: 1;
+    inline-size: 100%;
+    max-inline-size: px2rem(500);
```

Comment:
is the 500px `max-inline-size` a repeated value in other components, or just this one?

Reply:
Just this one it seems :)